cl/users/tests.py

```diff
@@ -1279,7 +1234,11 @@ def test_sending_to_banned_email(self) -> None:
         stored_email = EmailSent.objects.all()
         self.assertEqual(stored_email.count(), 0)
 
-    def test_sending_email_within_back_off(self) -> None:
+    @mock.patch(
+        "cl.users.email_handlers.send_failed_email",
+        side_effect=mock_send_failed_email,
```

Comment:
Why not use a lambda here and elsewhere: `side_effect=lambda x: None,`

The advantage is that this would put all the logic right here, so you wouldn't have to go look up what `mock_send_failed_email` did.

Reply:
Changed!